manifests/params.pp

```diff
@@ -50,7 +50,7 @@
   $influxdb_port                     = '8086'
   $influxdb_database                 = 'statsd'
   $influxdb_username                 = 'root'
-  $influxdb_password                 = 'root'
+  $influxdb_password                 = hiera('influx_db_pwd')
```

Comment:
dont use the `hiera` function here either use automatic parameter lookup which requires at least puppet 4.10.  or inherit from the params file

Reply:
Thanks for the feedback. I have updated my changes.